src/lib/y2storage/proposal_settings.rb

```diff
@@ -105,6 +105,15 @@ class ProposalSettings
     #   the root filesystem
     attr_accessor :subvolumes
 
+    # @note :ng format
```

Comment:
This is not legacy or ng format. Is simply not a proposal setting. :-)  But in any case, it has always been there.